helm-util/README.md

```diff
@@ -1,44 +1,114 @@
-# helm-util
+---
+permalink: /
+---
 
-Provides utilities for using helm in jsonnet.
+# package helm-util
 
-## Helmraiser
+```jsonnet
+local helm-util = import "github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet"
+```
 
-Helmraiser refers to a Go package that provides helmTemplate native function through Tanka for generating a Jsonnet
-library from any Helm chart. The goal of this function is to keep track of upstream changes in a Helm chart while
-retaining Jsonnet changes (read: mixins) on top of them.
+Package `helm-util` allows to consume Helm Charts as plain Jsonnet resources.
 
-## Example usage
+### Usage
 
-```
-jb install github.com/grafana/jsonnet-libs/helm-util
-```
+> **Warning:** [Functionality required](#internals) by this library is still
+> experimental and may break at any time.
 
-```
-local helm = import 'github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet';
+The [`helm.template`](#fn-helmtemplate) function to converts a Helm Chart into Jsonnet objects,
+to be consumed by tools like [Grafana Tanka](https://tanka.dev).
+
+Helm Charts are required to be available on the local file system and are
+resolved relative to the file that calls `helm.template`:
+
+```jsonnet
+local helm = (import 'github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet').new(std.thisFile);
 
 {
-  version:: 'v0.16.0',
-
-  values:: {
-    installCRDs: true,
-    global: {
-      podSecurityPolicy: {
-        enabled: true,
-        useAppArmor: false,
-      },
-    },
-  },
-
-  chart: helm.template(
-    name='cert-manager',
-    chart='jetstack/cert-manager',
-    conf={
-      values: $.values,
-      flags: [
-        '--version=%s' % $.version,
-      ],
+  // render the Grafana Chart, set namespace to "test"
+  grafana: helm.template('grafana', './charts/grafana', {
+    values: {
+      persistence: { enabled: true },
+      plugins: ['grafana-clock-panel'],
     },
-  ),
+    namespace: 'test',
+  }),
 }
+
+```
+
+### Chart Management
+
+To simplify Chart vendoring, Tanka includes a special tool at `tk tool vendor`:
+
+```bash
+# create a chartfile.yaml, similar to jsonnetfile.json
+$ tk tool charts init
+
+# install the Grafana Chart to ./charts/grafana
+$ tk tool charts add stable/grafana@5.5.5
 ```
+
+### Internals
+
+The functionality of `helm-util` is based on the `helm template` command.
+Because Jsonnet does not support executing arbitrary command for [good
+reasons](https://jsonnet.org/ref/language.html#independence-from-the-environment-hermeticity),
+a different way was required.
```

Comment:
A different way to what?? You need to say.

Reply:
A different way to executing arbitrary commands?